recipe/bld.bat

```diff
@@ -12,6 +12,7 @@ cmake -GNinja ^
       -D PNG_PNG_INCLUDE_DIR=%LIBRARY_INC% ^
       -D ZLIB_LIBRARY=%LIBRARY_LIB%\zlib.lib ^
       -D ZLIB_INCLUDE_DIR=%LIBRARY_INC% ^
+      -D BUILD_TESTING=ON ^
```

Comment:
As build tests are enabled, does that mean this is packaging them up as well?

Reply:
No. Tests are never installed. Installing stuff in cmake is an explicit action that has to be done in the cmake file.